spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -257,6 +257,22 @@ Reloading the Public Key from the location at runtime as well as the frequency o
 reloading is beyond the scope of this specification and any such feature should be
 considered vendor-specific.
 
+#### `mp.jwt.token.header`
+
+The `mp.jwt.token.header` configuration property allows to set up the header which is expected to contain a JWT token.
+
+This configuration can either be `Authorization` (default), or `Cookie`. If a different configuration value is supplied,
+the configuration will be ignored and fallback to the default `Authorization`.
```

Comment:
I believe it should fail with `DeploymentException` as otherwise MP JWT will be checking not what the application expects